https://github.com/nick-dodonov/lnav/commit/395c360c43820f5ea2238a8413280e053bc23383

From 395c360c43820f5ea2238a8413280e053bc23383 Mon Sep 17 00:00:00 2001
From: Nick Dodonov <nick.dodonov@gmail.com>
Date: Thu, 10 Oct 2024 11:14:11 +0300
Subject: [PATCH] [patch] prepare patch for `ebuild lnav-0.12.2.ebuild test` -
 disable same tests as lnav-0.11.0-disable-tests.patch

Original comment:

From 70231dce88cf47f2f3e2b396fe0623ea74eeadfc Mon Sep 17 00:00:00 2001
From: Randy Barlow <randy@electronsweatshop.com>
Date: Sat, 3 Sep 2022 18:13:08 -0400
Subject: [PATCH] Disable some flaky tests

Certain tests commented out for stability reasons:
 * listview: Fails when building inside sandbox
   https://github.com/tstack/lnav/issues/1039
 * mvattrline: Fails on some testers boxes. Documented here:
   https://github.com/tstack/lnav/issues/911
 * logfile: Flaky. Fails some times and not others. i.e. Race condition
 * tui: Can hang the build

Signed-off-by: Randy Barlow <randy@electronsweatshop.com>
---
 test/Makefile.am | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/test/Makefile.am b/test/Makefile.am
index f7d83f78..a6c252f8 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -210,10 +210,7 @@ dist_noinst_SCRIPTS = \
 	test_grep_proc.sh \
 	test_json_format.sh \
 	test_line_buffer.sh \
-	test_listview.sh \
-	test_logfile.sh \
 	test_meta.sh \
-	test_mvwattrline.sh \
 	test_prql.sh \
 	test_regex101.sh \
 	test_remote.sh \
@@ -234,7 +231,6 @@ dist_noinst_SCRIPTS = \
 	test_sql_xml_func.sh \
 	test_sql_yaml_func.sh \
 	test_text_file.sh \
-	test_tui.sh \
 	test_view_colors.sh \
 	test_vt52_curses.sh \
 	test_pretty_print.sh
@@ -480,15 +476,12 @@ TESTS = \
 	test_cmds.sh \
 	test_config.sh \
 	test_events.sh \
-	test_listview.sh \
 	test_meta.sh \
-	test_mvwattrline.sh \
 	test_gantt.sh \
 	test_grep_proc.sh \
 	test_grep_proc2 \
 	test_json_format.sh \
 	test_log_accel \
-	test_logfile.sh \
 	test_reltime \
 	test_scripts.sh \
 	test_sessions.sh \
@@ -508,7 +501,6 @@ TESTS = \
 	test_sql_yaml_func.sh \
 	test_text_anonymizer \
 	test_text_file.sh \
-	test_tui.sh \
 	test_data_parser.sh \
 	test_pretty_print.sh \
     test_view_colors.sh \
-- 
2.47.0

